starlight/views.py

```diff
@@ -12,15 +12,32 @@
 from django.shortcuts import render
 from django.views.decorators.csrf import csrf_protect
 
-from starlight.admin import MyUserCreationForm
-from starlight.forms import EditForm, CompetencyForm, EmployeeForm
-from starlight.models import Skill, Employee, Competency
+from starlight.forms import EditForm, CompetencyForm, EmployeeForm, TeamForm, FilterTeamForm
+from starlight.models import Skill, Employee, Competency, Team
 
 
 def home(request):
+    if request.method == 'POST':
+        form = FilterTeamForm(request.POST)
+        if form.is_valid():
+            filter_team = form.cleaned_data['name']
+            filter_name = filter_team.name
+        else:
+            filter_name = None
+    else:
+        form = FilterTeamForm()
+        filter_name = None
+
     skills = Skill.objects.order_by('name')
-    employees = Employee.objects.all()
-    return render(request, 'views/home.html', {'skills': skills, 'employees': employees, 'viewgroup': 'home'})
+
+    if filter_name and not filter_name == 'All':
```

Comment:
If our filter's `<option>` value is an empty string, here we can simply check `if filter_name`.